client/subcomponents/News/News.jsx

```diff
@@ -5,13 +5,33 @@ export default function News({ news }) {
   const createTime = moment(news.createdOn, 'DD/MM/YYYY').fromNow()
 
   return (
-    <article>
-      <h2>{news.title}</h2>
-      <h2>
-        By {news.firstName} {news.lastName}:
-      </h2>
-      <p>{news.content}</p>
-      <p>{createTime}</p>
-    </article>
+    <div className="relative rounded-3xl border-3 shadow-xl">
```

Comment:
do some research into "semantic html" and see where you might be able to add more to this component. you've already put it to great use here already, but overall less `<div>` use and more specifically labeled elements means this code base will be more readable and more accessible for users :) [semantic html guide](https://www.freecodecamp.org/news/semantic-html5-elements/#:~:text=Semantic%20HTML%20elements%20are%20those,content%20that%20is%20inside%20them.)

Reply:
Less divs, more sections. Articles too!  
Snappy for the screen readers. Hope you like it. Thanks Johnny.